migrations/20201116042500-update-ambulances-unique.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `DELETE FROM ambulances WHERE "id" NOT IN (SELECT * FROM (SELECT MIN("id") FROM ambulances GROUP BY "vehicleNumber") x)`,
```

Comment:
Just want to make sure I understand this query: you are grouping a database by vehicle numbers and then finding the minimum ID within each group of vehicle numbers. If the ambulances are not unique, we will be missing a particular ambulance ID, which indicates a duplicate?

Reply:
Yeah, you got it. The ambulance table is grouped into rows that have the same `vehicleNumber` and the minimum ID is selected from each row. At this point, we have a table of the ambulances that are not duplicates. Finally, we delete all ambulances whose IDs are not in this table (duplicates).

After discussing with @petorsong, we decided to only keep the lowest ID of a duplicate ambulance/hospital. 

An alternative could have been to check the associations between the hospital/ambulances and other entities before deleting them - however, I believe (?) that we decided that there would be no clean approach to deleting duplicates and went with this option.

Reply:
yeah basically we're hoping that this shouldn't affect production/staging since there's not much data on those environments and even less complex association hierarchies - if it breaks on dev envs i'll just help whoever fix it